Sources/Types+Media.swift

```diff
@@ -95,7 +104,7 @@ extension Media {
     public var bitRate: Int?
     
     /// Type of media
-    public var contentType: MediaType
+    public var contentType: ContentMediaType
```

Comment:
I think this should probably just conform to [`UTType`](https://developer.apple.com/documentation/uniformtypeidentifiers/uttype)!

Reply:
Wow...first time I saw this existed 😅 Seems convenient. It covers all "standard" extensions/types basically?